modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -463,18 +509,32 @@ The following table lists the known issues and, if fixed, the version in which t
 |===
 | Known issues | Fixed in
 
+| Using the legacy procedure link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_vector_createnodeindex[`db.index.vector.createNodeIndex`] on a database created prior to Neo4j 5.11 in Neo4j 5.18 and later, before the database has been upgraded, can cause an error.
```

Comment:
It's not the most clear I agree.

Ok, I will expand the details here, and lets see if we can find a simple and clear way to set the scene.

There are 3 versions to think of.
* The binary version, the version of the JAR downloaded/compiled.
* The runtime version, the version the DBMS is running as.
* The kernel version, the version of the database

During an upgrade (think cluster), we are switching out binaries to newer ones (binary). During that time, the machines on the new binaries cannot use the new stuff, they have to talk in the older version that the others are on (runtime). Once all machines are on the new binary, we let it know it can update to the newer runtime (this was an explicit procedure call, but I think recent clusters can figure this out now, not sure). The first write operation to a database will upgrade the database to the new version (kernel). So even though the database "older", because the "runtime" is newer, we can use the new things as that operation will update the database.

This also happens in community edition.

---

Ok with that context.

Someone creates a Neo4j database on a version prior to 5.11. Lets say on a binary version of 5.7 for an example.
They populate the database with their data. Perhaps they also fill it with pregenerated embeddings.

They want to use Neo4j vector stuff, so they update their binaries. They pick a new version (5.18 or newer) lets say they grab 5.22 as that's the newest.

The start the db, and they may optionally do a few read queries to check things are still working, but do not do a write. The first write operation they do is to create a vector index using the legacy procedure. _*BOOM*_ error.

Reply:
How about something more like:

```suggestion
| The creation of a vector index using the legacy procedure link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/procedures/#procedure_db_index_vector_createnodeindex[`db.index.vector.createNodeIndex`] may fail with an error in Neo4j 5.18 and later if the database was last written to with a version prior to Neo4j 5.11, and the legacy procedure is the first write operations.
```

Reply:
I like it. Maybe the last bit could say
"...and later if the database was last written to with a version prior to Neo4j 5.11, and the legacy procedure is the first write operation used on the newer version."